packages/hello-world/__tests__/hello-world.test.ts

```diff
@@ -0,0 +1,5 @@
+"use strict";
```

Comment:
Test files in TS? Oh my

Reply:
💥 

We don't actually need to do this, however I find the imports and type-completion to be a bit simpler.